packages/syntax-core/src/Box/Box.module.css

```diff
@@ -0,0 +1,359 @@
+.box {
+  box-sizing: border-box;
+}
+
+.flexWrap {
+  flex-wrap: wrap;
+}
+
+/* Align items */
+.alignItemscenter {
```

Comment:
Why is the last word lower case for all of these? I haven't seen this pattern before